### ISSUE TITLE
Remove ESP8266 code paths

### ISSUE THREAD

**Issue by mairas** [SignalK/SensESP#457]
Version 2 will no longer support ESP8266; the conditional code paths should be removed.

Also remove the 8266 specific dependencies from library.json and platformio.ini!

**Comment by ba58smith** [comment]
Bravo!

**Comment by JohnySeven** [comment]
Great! But I would leave at least one def something like

```
#if ESP8266
#error "ESP8266 isn't supported by SensESP v2"
#endif
```
Just to be clear when somebody forgot to specify v1 in platformio.ini, this would stop him right there.

**Comment by mairas** [comment]
> Great! But I would leave at least one def something like
> 
> ```
> #if ESP8266
> #error "ESP8266 isn't supported by SensESP v2"
> #endif
> ```
> 
> Just to be clear when somebody forgot to specify v1 in platformio.ini, this would stop him right there.

This is the way to go! And furthermore, the error message should link to instructions on how to peg SensESP to v1 to keep existing code from breaking.

**Comment by ba58smith** [comment]
@mairas, when you're almost finished with all the V2 changes, let me know, and I'll start working on redoing the documentation. (Yee ha! My favorite thing!)

**Comment by mairas** [comment]
This time, I'd like to participate in the documentation work as well!

**Comment by ba58smith** [comment]
Be my guest! Feel free to write any of it that you'd like, or to direct how you'd like for it to be organized, and I can handle the details.

**Comment by mairas** [comment]
Fixed in #478. Why didn't that PR close this issue automatically?
